view/components/features/login-form.tsx

```diff
@@ -26,6 +26,35 @@ export interface LoginFormProps {
 export function LoginForm({ ...props }: LoginFormProps) {
   const { t } = useTranslation();
   const [showPassword, setShowPassword] = useState(false);
+  const [emailError,setEmailError] = useState('');
```

Comment:
i hope these state updates will be rendered without an effect trigger requirement.